### DISCUSSION TITLE
【剧荒】网友们有无推荐喜剧

### DISCUSSION THREAD

**Discussion by ltsraining** [share121/inter-knot#5316] [其他]
习惯一边游戏一边放个剧看，最近惊觉已经很久没看新剧了，都是在一遍遍刷看过的剧，觉得确实要着手发扬列车开拓意志，开展调查工作，挖掘先进思想……（划掉    
所以请网友们给我推荐一些好笑的情景喜剧吧！或者好玩好看有趣的剧也行，主要觉得看喜剧会让心情很好所以希望是喜剧，不是也可以觉得好看就发发发，所有分类我基本都不挑也没有雷，我把我觉得好看的喜剧（好笑的剧？）在下面列个推荐--安利--，想起来哪个写哪个，让我们一起 make 情景喜剧 great again   
      
①老友记，最广为人知的情景喜剧吧，六个好朋友的故事，现实中怎么不让我有这样的一群朋友（含恨而终   
②神探夏洛克，其实不是情景喜剧，但不好笑吗（贴脸猫猫）家喻户晓的福尔摩斯现代版   
③欢乐一家亲，Frasier我单方面宣布演技最好的是Eddie（划掉），是前传干杯酒吧里心理医生的一家的故事，就爱看老弗生气，现在后传也已经出到第二季第二集了，大家都老了……，还有干杯酒吧资源真的好难找如果谁有完整中字资源的话球球   
④神烦警探，是警局的故事但好像不算刑侦剧……99的笑点真的死戳我的笑点，其实最想问有没有和99相似的剧   
⑤爆笑女警，完全无厘头，建议不带脑子看，能get到笑点的话是真的很好笑   
⑥老爸老妈罗曼史，其实是--老爸和罗宾阿姨罗曼史--
，也是一群朋友的喜剧，叙述方法很出彩   
⑦好汉两个半，小时候一直听说但看名字一直以为是战争片（？）其实是俩兄弟带孩子   
⑧威尔和格蕾丝，同性恋里程碑的剧，直女和gay一起看孩子（并不是！）   
⑨生活大爆炸，知名度也很能打，科技宅改变世界（划掉   
⑩小谢尔顿，楼上tbbt的衍生剧，耳朵的小时候   
破产姐妹，破产的富家女被黑老大包养的故事（也不是！   
①①燃情克利夫兰，明星 修眉师 小说家离开大城市定居小城市遇到嘴毒门房的故事   
①②天生冤家，邋遢体育主播被迫和强迫症合住的故事，钱钱翻拍的剧，再次惋惜马修……老友记真的最喜欢钱钱，小知识片头曲是老友记第一季有一集大家一起哼的阿卡贝拉   
①③老爸有招，老爸从熊孩子治到一切，老友记Joey演的剧   
①④查莉成长日记，一家人拍给小查莉的视频日记，话说查莉真的可以说是在剧组长大的吧   
①⑤灵异妙探，我通灵师今天就要让你知道我通灵术的威力！（其实是假的，推理装通灵，题材类似有超感神探
①⑥妙警贼探，影版是小李子的猫鼠游戏，骗子从良不从良从良不从良反复横跳的故事
极品基老伴，gay的老年生活，常看英剧的话全是熟脸
①⑦古宅老友记，继承祖屋却获得能看见鬼的能力，从原始人尖叫到知名政客，其实后面大家都很好，已经结局了很sad，英剧为什么都这么短小……
①⑧鬼屋欢乐送，楼上古宅老友记的美版，人员和设定都有所变动，基本可以当新片来看了
①⑨我是大哥大，黄毛和海胆头的“从今天起要做不良少年”之路。想起来日剧了，感觉应该有很多好笑的！大家也可以推推！   
②⑩勇者义彦的故事，确实是勇者斗魔王的故事，确实啊确实，谁说不是呢，有好几部，B站可看
②①武林外传，这个感觉就不用介绍了，还有个后传是叫龙门镖局还是龙门客栈来着   
②②家有儿女，不用介绍+1，但说起来其实没有完整看完过，都是在电视上东看一点西看一点   
②③万万没想到，王大锤三个字出来感觉也不用介绍了，还有同团队的老板驾到和什么来着，当年的万合天宜真的好能打，现在也很能打，扬名立万也很好看   
   
实在想不起来了，安利到此为止接下来坐等投喂……以及为什么我的符号表里只能打到⑩   
   
写给没看过欧美剧的网友：欧美剧的话前几集一般是试播集，人物在铺开和摸索中，可能没有非常精彩，后面熟悉人物关系之后展开的故事和笑点就很多了。虽然但是感觉没看过欧美剧的网友应该也不会坚持看到这里的……写上备用   
   
对了以上的大部分剧人人视频能看，iOS大概没有人人，B站也有一些，其他的可能要网盘见，如果需要的话可以找我要资源（不一定有如果人人能看的我就人人看了），大家有其他稳定的看剧的APP或者网站的话也可以推荐下，再次感谢各位网友啦！

**Comment by manyshoes** [comment]
是，大臣，以前半夜睡不着就躺在椅子上看这个<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Comment by RWHwaxili** [comment]
喜剧吗 可以看看小身材大危机

**Reply by ltsraining** [reply]
感觉只有在六分街能找到资源啊，我个人因为体质原因大概去不到六分街，R老师可以代我租一盘录像带吗
![mmexportb0cc6862739ba57c23bb59a319b9df14_1726165795541](https://github.com/user-attachments/assets/bf95d1d9-e087-4c13-8517-131f874971c0)

**Comment by KELERSAMA** [comment]
《大赢家》吧 感觉挺好看的

**Reply by ltsraining** [reply]
搜了下，是银行演习那个电影吗，记下了

**Comment by ltsraining** [comment]
我去看 是，大臣 回来了
前两集：说的什么好难懂 英剧语速+政治术语=流心浆糊脑袋（我的
后面搞明白设定就好看了，第四集就反击成功笑死我了我差点以为要讽刺到底全职傀儡

**Reply by manyshoes** [reply]
一开始是被汉弗莱长难句和那个经典五步法吸引入坑的，看下去之后才发现太有意思了<img src="https://bbs-static.miyoushe.com/static/2023/04/28/0c624a8915dd9b24093cf630bded5748_7288657913684710277.png" alt="米仔_拜年" width="75" >
